mindsdb/integrations/handlers/anyscale_endpoints_handler/anyscale_endpoints_handler.py

```diff
@@ -89,7 +90,7 @@ def describe(self, attribute: Optional[str] = None) -> pd.DataFrame:
     def _set_models(self, args):
         if 'api_key' in args:
             args['openai_api_key'] = args['api_key']  # remove this once #7496 is fixed
-        client = openai.OpenAI(api_key=get_api_key('openai', args, self.engine_storage), base_url=ANYSCALE_API_BASE)
+        client = self._get_client(get_api_key('openai', args, self.engine_storage))
```

Comment:
Use this instead as base class also calls the method, so we overwrite with `ANYSCALE_API_BASE`.